### PR TITLE
Improve documentation and init

### PR DESCRIPTION
**Pull request by Zefinder**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -52,4 +77,4 @@ then
         echo "Installing minicom..."
         sudo apt install minicom
     fi
-fi
\ No newline at end of file
+fi
```
Comment (Zefinder) [inline init.sh:80]: Would it be a good idea to add the installation of the libs in the readme's prerequisite section in the init file?

Comment (GOAguiar99) [reply init.sh:91]: FIxed

**Review by GOAguiar99 — APPROVED** [review]

File: bao-hypervisor
Comment (Zefinder) [inline bao-hypervisor:1]: If it has been updated, please make a pull request there!

File: freertos-bao-fpmc
Comment (Zefinder) [inline freertos-bao-fpmc:1]: Same, make a pull request there... Else remove change from PR

**Pull request merged**